docs/source/gcloud_topo.rst

```diff
@@ -8,14 +8,51 @@ Gcloud Topologies
 Google Cloud Topologies
 ```````````````````````
 
-Google cloud topologies . 
+.. code-block:: bash
```

Comment:
Would this be better with code-block:: yaml?

http://symfony.com/doc/current/contributing/documentation/format.html#syntax-highlighting